docs/source/en/model_doc/mobilenet_v1.md

```diff
@@ -14,43 +14,102 @@ rendered properly in your Markdown viewer.
+# Preprocess image
+inputs = processor(images=image, return_tensors="pt")
 
-- One can use [`MobileNetV1ImageProcessor`] to prepare images for the model.
+# Perform inference
+with torch.no_grad():
+    outputs = model(**inputs)
+    logits = outputs.logits
 
-- The available image classification checkpoints are pre-trained on [ImageNet-1k](https://huggingface.co/datasets/imagenet-1k) (also referred to as ILSVRC 2012, a collection of 1.3 million images and 1,000 classes). However, the model predicts 1001 classes: the 1000 classes from ImageNet plus an extra “background” class (index 0).
+# Post-process to get labels
+predicted_label_id = logits.argmax(-1).item()
+predicted_label = model.config.id2label[predicted_label_id]
+print(predicted_label)
+# Example output: tabby, tabby cat
+```
 
-- The original TensorFlow checkpoints use different padding rules than PyTorch, requiring the model to determine the padding amount at inference time, since this depends on the input image size. To use native PyTorch padding behavior, create a [`MobileNetV1Config`] with `tf_padding = False`.
+</hfoption>
+</hfoptions>
 
-Unsupported features:
+<!-- Quantization - Not applicable -->
+<!-- Attention Visualization - Not applicable for this model type -->
 
-- The [`MobileNetV1Model`] outputs a globally pooled version of the last hidden state. In the original model it is possible to use a 7x7 average pooling layer with stride 2 instead of global pooling. For larger inputs, this gives a pooled output that is larger than 1x1 pixel. The HuggingFace implementation does not support this.
 
-- It is currently not possible to specify an `output_stride`. For smaller output strides, the original model invokes dilated convolution to prevent the spatial resolution from being reduced further. The output stride of the HuggingFace model is always 32.
+## Notes
 
-- The original TensorFlow checkpoints include quantized models. We do not support these models as they include additional "FakeQuantization" operations to unquantize the weights.
+-   **Checkpoint Naming:** Checkpoints often follow the pattern `mobilenet_v1_{depth_multiplier}_{resolution}`, like `mobilenet_v1_1.0_224`.
+-   **Variable Input Size:** While trained on fixed sizes (e.g., 224x224), the model architecture works with images of different sizes (minimum 32x32). The [`MobileNetV1ImageProcessor`] handles the necessary preprocessing.
+-   **1001 Classes:** Models pretrained on ImageNet-1k output 1001 classes. Index 0 is a "background" class, and indices 1-1000 correspond to the standard ImageNet classes.
+-   **Padding Differences:** Original TensorFlow checkpoints had dynamic padding behavior based on input size. The Hugging Face PyTorch implementation uses standard, static padding by default. To enable dynamic padding (matching TensorFlow behavior), instantiate the configuration with `tf_padding=True`.
```

Comment:
Nice example below!

```suggestion
-   The original TensorFlow checkpoints determines the padding amount at inference because it depends on the input image size. To use the native PyTorch padding behavior, set `tf_padding=False` in [`MobileNetV1Config`].
```